searx/searx
# New files engine: Prowlarr

## What does this PR do?

Gives the user the possibility to search their own prowlarr instances.

Info: https://wiki.servarr.com/en/prowlarr
Github: https://github.com/Prowlarr/Prowlarr

## Why is this change important?

Prowlarr searchs multiple upstream search providers, thus allows to use that functionality through searx.

## How to test this PR locally?

* Run sudo make run
* Go to /preferences
* Enable prowlarr search engine
* Make some search
* Assert that it works as expected

## Author's checklist

This uses the prowlarr search API; it does not support pagination.

## Related issues

(none)
